src/App/app-main.js

```diff
@@ -63,6 +63,9 @@ export class AppTemplate extends Component {
       {
         className: 'pub', type: 'button', iconClass: 'fas fa-beer', link: '', name: 'Pub Songs',
       },
+      {
+        className: 'shop', type: 'link', iconClass: 'fas fa-shopping-cart', link: '/music/shop', name: 'Web Jam Shop',
```

Comment:
this needs to be /shop

not /music/shop

make a root page at / that is just a temp page for now and then fix the other route and subroutes please